### DISCUSSION TITLE
关于 Callout / Admonition 的支持

### DISCUSSION THREAD

**Discussion by xuejiaW** [CaiJimmy/hugo-theme-stack#675] [Q&A] [answered]
目前模板支持 Callout / Admonition 吗

在模板的说明文档中，看到有使用 note / warning Callout，但在博客的 Post 中没有找到能正确渲染的方式

尝试了 的形式

> [!note]
> This admonition won't have a title!

**Comment by CaiJimmy** [comment, marked as answer]
> 目前模板支持 Callout / Admonition 吗

不支持。

> 在模板的说明文档中，看到有使用 note / warning Callout

是文档主题的功能（Vuepress）。

**Reply by kamasylvia** [reply]
有支持 Admonition 的计划吗？我看有人在 stack 上[自定义 Admonition](https://blog.ty-wu.com/posts/hugoadmonitionshortcode/)了还挺简单的。
